### ISSUE TITLE
Shaka throws MANIFEST.RESTRICTIONS_CANNOT_BE_MET on Tizen 2016

### ISSUE THREAD

**Issue by chrisfillmore** [google/shaka-player#884]
On Tizen 2016 model TV's, Shaka dispatches `MANIFEST.RESTRICTIONS_CANNOT_BE_MET`, from `Player#chooseStreams_`. There aren't any playable variants because it uses the webkit MediaKeys polyfill, which uses the fake KEY_ID_:
```
// Construct fake key ID.  This is not done at load-time to avoid exceptions
// on unsupported browsers.  This particular fake key ID was suggested in
// w3c/encrypted-media#32.
PatchedMediaKeysWebkit.MediaKeyStatusMap.KEY_ID_ =
    (new Uint8Array([0])).buffer;
```
Thus none of the `stream.keyId` keys exist in the `keyStatusMap`  in `Player#onKeyStatus_`, here:
```
var keyStatus = keyStatusMap[stream.keyId];
variant.allowedByKeySystem = !!keyStatus && restrictedStatuses.indexOf(keyStatus) < 0;
```
...and thus the variant is restricted.

But playback proceeds anyway. Can you share any more information about the meaning of this error? I didn't dig further into what happens when the player dispatches this error, so I'm not sure if this is expected behaviour or not.

Thanks!

**Comment by joeyparrish** [comment]
I believe this is related to #890 and #891.  In all three issues, there is a lack of key status information.  We will try to address #890 first and see how well that work applies to Tizen.

**Comment by joeyparrish** [comment]
The fix for this has been cherry-picked and will be released in v2.1.5.
